gui/src/app/state/settings/mod.rs

```diff
@@ -84,6 +84,10 @@ impl State for SettingsState {
                     .map(|s| s.reload(daemon, wallet))
                     .unwrap_or_else(Command::none)
             }
+            Message::WalletUpdated(Ok(wallet)) => {
```

Comment:
The message must also be passed to the self.setting update, otherwise the "Updated" green text is not shown and the button keeps the "Processing" state.

Reply:
done